code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -317,7 +280,7 @@
 	anchored = TRUE
 
 	if(isspaceturf(target_turf)) //If we are fixing an area not part of pure space, it is
-		visible_message("<span class='notice'>[targetdirection ? "[src] begins installing a bridge plating." : "[src] begins to repair the hole."] </span>")
+		visible_message("<span class='notice'>["[src] begins to repair the hole."] </span>")
```

Comment:
```suggestion
		visible_message("<span class='notice'>[src] begins to repair the hole.</span>")
```

Reply:
commiting, but for learning purposes, what is the pupose of the change